src/android/toga_android/widgets/label.py

```diff
@@ -18,10 +18,16 @@ def set_text(self, value):
         self.native.setText(value)
 
     def rehint(self):
+        # With the Android TextView, we use `measureText()` to compute the width
+        # of the text -- calling `measure()` seems to ignore the text, perhaps
+        # because the TextView is willing to truncate the text. We do use
+        # height information from `measure()`, which seems fine.
```

Comment:
Annoying, but makes at least *some* sense.

Reply:
The thing I just figured out right now is that I think the problem is I'm constraining neither width nor height.

I think I can do this a different way, which would also take two passes, but would be a little more defensible.

Phase 1: Ask TextView what its height is, given no constraint on height or width.

Phase 2: Ask TextView what its width is, given a constraint of being no higher than the height we found in phase 1.

I'll test this tomorrow. If it works, I prefer it, since I prefer to deal with just `.measure()` and let the rest be an implementation detail of the poor Android widget.

Reply:
It does work :) maybe Android is not as bad as I thought.